.github/workflows/stale.yml

```
@@ -0,0 +1,42 @@
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write  
+  
+jobs:
+  stale:
+    name: Process Stale Issues and PRs 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)